src/android/device.rs

```diff
@@ -60,6 +232,9 @@ pub enum RunError {
     StartFailed(bossy::Error),
     WakeScreenFailed(bossy::Error),
     LogcatFailed(bossy::Error),
+    BundletoolInstallFailed(BundletoolInstallError),
+    AabBuildError(AabBuildError),
```

Comment:
Error variant names should be suffixed with `Failed`, while the error type itself has the `Error` suffix